transformer_engine/common/common.cu

```diff
@@ -135,3 +135,19 @@ bool is_supported_by_CC_100() {
 }
 
 }  // namespace transformer_engine
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+bool nvte_is_supported_nontn_fp8_gemm() {
```

Comment:
This name feels awkward to me. How about:
```suggestion
bool nvte_is_non_tn_fp8_gemm_supported() {
```
This matches the PyTorch function: https://github.com/NVIDIA/TransformerEngine/blob/04642bf8910da5b800b10d98e32ad1a310e106cd/transformer_engine/pytorch/utils.py#L253
Alternatively, we could name it `nvte_is_only_tn_fp8_gemm_supported`, so we don't not avoid multiple negatives like `!nvte_is_non_tn_fp8_gemm_supported()`.

Reply:
I changed the name to suggested. Also, I feel like a better version of this function should be `get_supported_gemm_types` and then in the code we could do `"<gemm_type>" in get_supported_gemm_types()`. Since this is a very specific use-case for now, maybe we could let this as is?